astropy/io/fits/tests/test_table.py

```diff
@@ -976,7 +975,7 @@ def test_add_data_by_rows(self):
         assert tbhdu.columns.columns[2].array[0] == ''
         assert (tbhdu.columns.columns[3].array[0] ==
                 np.array([0., 0., 0., 0., 0.], dtype=np.float32)).all()
-        assert tbhdu.columns.columns[4].array[0] == True
+        assert tbhdu.columns.columns[4].array[0] == True  # nopep8
```

Comment:
There is a mixture of np.bool_ and bool types here where np.bool_ is not a subclass of bool, so it gave a failuer with ``is True``, though it wasn't clear to me where and why it became an np.True_ rather than a True. 
Thus left it unchanged.